lib/server.js

```diff
@@ -102,9 +102,9 @@ function parsePOST(req) {
 
 function logResponse(opts, response, startTime) {
     const latency = Date.now() - startTime;
-    let logLevel = 'trace/request';
+    let logLevel = 'info/request';
```

Comment:
The the logger implicitly adds information about the root request, including all the interesting headers. See [this logstash query](https://logstash.wikimedia.org/app/kibana#/dashboard/restbase?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:now-15m,mode:quick,to:now))&_a=(filters:!(('$state':(store:appState),meta:(alias:!n,disabled:!f,index:'logstash-*',key:_type,negate:!f,value:restbase),query:(match:(_type:(query:restbase))))),options:(darkTheme:!f),panels:!((col:1,id:Events-Over-Time,panelIndex:1,row:1,size_x:9,size_y:2,type:visualization),(col:1,columns:!(type,level,wiki,host,message),id:Default-Events-List,panelIndex:3,row:3,size_x:12,size_y:17,sort:!('@timestamp',desc),type:search),(col:10,id:Events-By-LevelPath,panelIndex:4,row:1,size_x:3,size_y:2,type:visualization)),query:(query_string:(analyze_wildcard:!t,query:%22trace%2Frequest%22)),title:restbase,uiState:(P-1:(vis:(legendOpen:!f))))) for current output.